diff --git a/paddle/fluid/operators/memcpy_h2d_op.h b/paddle/fluid/operators/memcpy_h2d_op.h
index 0d73142607..f99004b434 100644
--- a/paddle/fluid/operators/memcpy_h2d_op.h
+++ b/paddle/fluid/operators/memcpy_h2d_op.h
@@ -44,6 +44,7 @@ class MemcpyH2DFunctor {
         static_cast<const platform::CUDADeviceContext *>(&dev_ctx_)->stream();
 #else
     auto stream = nullptr;
+    (void)stream;
 #endif
     out_tensor.mutable_data(
         dev_ctx_.GetPlace(), lod_tensor.dtype(),
